src/Compilers/CSharp/Test/Semantic/Semantics/NativeIntTests.cs

```diff
@@ -15,6 +15,8 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Semantics
 {
     public class NativeIntTests : CSharpTestBase
```

Comment:
nit: not related to this PR: should there be a compiler feature on the test type? We've traditionally done that for new features.